src/ocean/core/Exception.d

```diff
@@ -100,9 +100,9 @@ unittest
 public template ReusableExceptionImplementation()
 {
     import ocean.transition;
-    static import ocean.text.convert.Integer_tango;
```

Comment:
is this really gone?

Reply:
not anymore, good catch

Reply:
Good thing we don't have any unittest so this doesn't go red ;-)

Reply:
Updated

Reply:
> Good thing we don't have any unittest so this doesn't go red ;-)

We have but there are other imports in same module that provide it so it kept being green :(